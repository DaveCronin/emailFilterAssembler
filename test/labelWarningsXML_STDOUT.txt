Warning, the label 'a/b' contains a '/' which you might be using to indicate nested/sub labels or folders. When uploading filters to gmail the importer treats the / as part of the label name, and creates the label as such.
It does not create a nested label with this information. However if a nested label aleardy exists, it will use it. So please create any nested labels first manually in gmail. Regular labels will be created automatically on import.
Sorry for the inconvenience this causes.

Warning, another potentially nested label 'c/b'

