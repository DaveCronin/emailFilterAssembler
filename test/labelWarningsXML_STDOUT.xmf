test = [label "a/b"]
test2 = [label "c/b"]

if {to "me"} -> test
if {to "you"} -> test2
if {to "him"} -> test
if {to "her"} -> test2