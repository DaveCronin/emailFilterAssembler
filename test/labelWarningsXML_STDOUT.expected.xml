<?xml version='1.0' encoding='UTF-8'?><feed xmlns='http://www.w3.org/2005/Atom' xmlns:apps='http://schemas.google.com/apps/2006'>
	<title>Mail Filters</title>
	<id>tag:mail.google.com,2008:filters:4366218717592519022</id>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='me'/>
		<apps:property name='label' value='a/b'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='you'/>
		<apps:property name='label' value='c/b'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='him'/>
		<apps:property name='label' value='a/b'/>
	</entry>
	<entry>
		<category term='filter'></category>
		<title>Mail Filter</title>
		<id>tag:mail.google.com,2008:filter:4366218717592519022</id>
		<apps:property name='to' value='her'/>
		<apps:property name='label' value='c/b'/>
	</entry>
</feed>